template/src/screens/DemoScreen.test.tsx

```diff
@@ -1,21 +1,37 @@
+import { comicMockParsed } from '__mocks__/fixtures'
 import React from 'react'
 import { TestIDs } from '@config/testIDs'
 import Routes from '@navigation/routes'
-import { RemoteDataStates } from '@utils/api'
-import { createNavigationProps, fireEvent, render } from '@utils/testing'
+import * as remoteComics from '@remote/comics'
+import { act, createNavigationProps, fireEvent, render } from '@utils/testing'
 import DemoScreen from './DemoScreen'
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 const navPropsMock = createNavigationProps() as any
 
+const mockUseLatestComicQuery = jest.spyOn(remoteComics, 'useLatestComicQuery').mockReturnValue({
+  isLoading: false,
+  data: undefined,
+} as ReturnType<typeof remoteComics.useLatestComicQuery>)
+
+jest.mock('@remote/auth', () => ({
+  useLogOutMutation: jest.fn().mockImplementation(() => ({
```

Comment:
Hooks with effects are mocked in component unit tests to avoid unfinished tests (promises running indefinitely).
It's a separate issue that existed here before migration to TanStack Query.